CMakeLists.txt

```diff
@@ -41,6 +41,8 @@ catkin_package(
     sensor_msgs
   DEPENDS
     console_bridge
+  LIBRARIES
+    fmt::fmt
```

Comment:
We do not need to export the fmt library, do we? (see http://wiki.ros.org/catkin/CMakeLists.txt)

Reply:
I think since our library links against fmt, others who link against us need this. tbh I am not 100% sure.

Reply:
Or this is only relevant if we exported our includes. Then we would not need to list any dependencies in `catkin_package()` at all.

Reply:
> I think since our library links against fmt, others who link against us need this. tbh I am not 100% sure.

If we want to do this we would need to do `DEPENDS fmt` inside of `catkin_package()`, but it might not be needed, since we already link it against our executable.
(see https://answers.ros.org/question/58498/what-is-the-purpose-of-catkin_depends/?answer=58593#post-id-58593 and the comment of mcamurri)

Reply:
I think you are right. https://github.com/PilzDE/psen_scan_v2/pull/110